cmd/timestamp-cli/app/timestamp.go

```diff
@@ -39,23 +39,37 @@ import (
 
 func addTimestampFlags(cmd *cobra.Command) {
 	cmd.Flags().Var(NewFlagValue(fileFlag, ""), "artifact", "path to an artifact to timestamp")
+	cmd.MarkFlagRequired("artifact") //nolint:errcheck
 	cmd.Flags().String("hash", "sha256", "hash algorithm to use - Valid values are sha256, sha384, and sha512")
+	cmd.MarkFlagRequired("hash") //nolint:errcheck
 	cmd.Flags().Bool("nonce", true, "specify a pseudo-random nonce in the request")
 	cmd.Flags().Bool("certificate", true, "if the timestamp response should contain a certificate chain")
 	cmd.Flags().Var(NewFlagValue(oidFlag, ""), "tsa-policy", "optional dotted OID notation for the policy that the TSA should use to create the response")
-
 	cmd.Flags().String("out", "response.tsr", "path to a file to write response.")
 }
 
-func validateTimestampFlags() error {
-	artifactStr := viper.GetString("artifact")
-	hashStr := viper.GetString("hash")
+type timestampCmdOutput struct {
+	Timestamp time.Time
+	Location  string
+}
 
-	if artifactStr == "" || hashStr == "" {
```

Comment:
Since this function is just checking that these flags had values set, I removed this function and use `cmd.MarkFlagRequired` instead.